docs/index.rst

```diff
@@ -11,32 +11,19 @@ Welcome to Renku!
 **Renku** is a software platform designed to enable reproducible and
 collaborative data science.
 
-The platform:
-
-* Allows data scientists to easily make their analyses reproducible by
-  automatically capturing the lineage of results in a Knowledge Graph
-
-* Turns parts of recorded data science pipelines into building blocks that can
-  be reused by others in their own analyses
-
-* Provides tools for working with the Knowledge Graph for the purposes of
-  traceability and auditability
+.. toctree::
+   :maxdepth: 2
 
-If you are a data scientist looking to get started with Renku, head to the
-:ref:`first_steps` tutorial for a quick look at how Renku can be used for a
-data science project. To read up on the concepts behind Renku, you may go
-straight to the :ref:`concepts` pages.
+   Introduction <introduction/index>
```

Comment:
I think this page is a little too minimal in this form. I would prefer to see a page of content explaining "Why Renku" and "What is Renku" on this page. There can be greater detail elsewhere, but the summary should be here.